okio/build.gradle

```diff
@@ -27,56 +31,61 @@ kotlin {
   mingwX64('winX64')
   sourceSets {
     commonMain {
-      kotlin.srcDir('src/main/kotlin')
+      kotlin.setSrcDirs(['src/main/kotlin'])
       dependencies {
         api deps.kotlin.stdLib.common
       }
     }
     commonTest {
-      kotlin.srcDir('src/test/kotlin')
+      kotlin.setSrcDirs(['src/test/kotlin'])
       dependencies {
         implementation deps.kotlin.test.common
         implementation deps.kotlin.test.annotations
       }
     }
     jvmMain {
-      kotlin.srcDir('jvm/src/main/java')
-      resources.srcDir('jvm/src/main/resources')
+      kotlin.setSrcDirs(['jvm/src/main/java'])
+      resources.setSrcDirs(['jvm/src/main/resources'])
       dependencies {
         api deps.kotlin.stdLib.jdk6
         compileOnly deps.animalSniffer.annotations
       }
     }
     jvmTest {
-      kotlin.srcDir('jvm/src/test/java')
+      kotlin.setSrcDirs(['jvm/src/test/java'])
       dependencies {
         implementation deps.test.junit
         implementation deps.test.assertj
         implementation deps.kotlin.test.jdk
       }
     }
     jsMain {
-      kotlin.srcDir('js/src/main/kotlin')
+      kotlin.setSrcDirs(['js/src/main/kotlin'])
       dependencies {
         api deps.kotlin.stdLib.js
       }
     }
     jsTest {
-      kotlin.srcDir('js/src/test/kotlin')
+      kotlin.setSrcDirs(['js/src/test/kotlin'])
       dependencies {
         implementation deps.kotlin.test.js
       }
     }
     nativeMain {
-      kotlin.srcDir('native/src/main/kotlin')
+      kotlin.setSrcDirs(['native/src/main/kotlin'])
+      dependsOn commonMain
```

Comment:
AFAIK these are redundant, platform specific builds depend on common by default.

Reply:
That's what I was thought as well, but I think I needed to add these so the navigation from expect to actual listed native code as one of the actual implementations. I can double check and remove if I ever figure out the `sample` project.

Reply:
Interesting! If that proves to be the case I think filing an issue will make sense.